upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -365,11 +365,8 @@ func (c *ApplyClusterCmd) Run() error {
 	case fi.CloudProviderVSphere:
 		{
 			vsphereCloud := cloud.(*vsphere.VSphereCloud)
-			region = vsphereCloud.Region
-
-			//if !AlphaAllowGCE.Enabled() {
-			//	return fmt.Errorf("GCE support is currently alpha, and is feature-gated.  export KOPS_FEATURE_FLAGS=AlphaAllowGCE")
-			//}
+			// TODO: map region with vCenter cluster, or datacenter, or datastore?
+			region = vsphereCloud.Cluster
```

Comment:
As long as all our code is enabled only with vsphere cloud provider flag then I dont see a need for feature flag.